dll/win32/shell32/CDefView.cpp

```diff
@@ -1675,7 +1676,7 @@ LRESULT CDefView::OnCreate(UINT uMsg, WPARAM wParam, LPARAM lParam, BOOL &bHandl
 
     // A folder is special if it is the Desktop folder,
     // a network folder, or a Control Panel folder
-    m_isParentFolderSpecial = _ILIsDesktop(m_pidlParent) || _ILIsNetHood(m_pidlParent)
+    m_isParentFolderSpecial = IsDesktop() || _ILIsNetHood(m_pidlParent)
```

Comment:
Why doesn't the pidl checking work there?

Reply:
Bullet point #2 in my PR text.

_ILIsDesktop is true for both the Desktop listview and when viewing the Desktop folder in an Explorer window. However, when viewing the Desktop folder in an Explorer window, you do want size information about files etc. (Windows does this as well).